map_server/src/main.cpp

```diff
@@ -186,10 +187,10 @@ class MapServer
       map_resp_.map.header.frame_id = frame_id;
       map_resp_.map.header.stamp.sec = now_sec;
       map_resp_.map.header.stamp.nanosec = now_nanosec;
-      printf("Read a %d X %d map @ %.3lf m/cell\n",
-               map_resp_.map.info.width,
-               map_resp_.map.info.height,
-               map_resp_.map.info.resolution);
+      RCUTILS_LOG_INFO("Read a %d X %d map @ %.3lf m/cell",
```

Comment:
Nit: please wrap after the open parenthesis so that we don't need to reindent the bloc when the macro name changes